src/components/plots/plot.ts

```diff
@@ -270,20 +272,24 @@ module Plottable {
       this._scales().forEach((scale) => scale._autoDomainIfAutomaticMode());
     }
 
-    private _updateExtentsForAttr(attr: string) {
+    protected _updateExtentsForAttr(attr: string) {
       var binding = this._attrBindings.get(attr);
       var accessor = binding.accessor;
+      var filter = this._attrFilters.get(attr);
```

Comment:
The comment about the _attrFIlters instance variable connects the implementation of filter here